src/Components/Details/Details.js

```diff
@@ -0,0 +1,62 @@
+import React from 'react'
+import './Details.css'
+import backArrow from './double-left-arrows.svg'
+
+const Details = ({ allMovies, selectedMovieID, hideSelection }) => {
+
+  const movieToDisplay = allMovies.find(movie => {
+    return movie.id === selectedMovieID
+  })
+
+  console.log('MOVIE >>> ', movieToDisplay)
```

Comment:
Oops! Let's make sure to remove the console.logs before pushing. (I am really bad about this!)